dbms/src/Common/TiFlashSecurity.h

```diff
@@ -272,18 +264,18 @@ class TiFlashSecurityConfig : public ConfigObject
         String new_key_path;
         if (config.has("security.ca_path"))
         {
-            new_ca_path = config.getString("security.ca_path");
-            miss_ca_path = false;
+            new_ca_path = Poco::trim(config.getString("security.ca_path"));
+            miss_ca_path = new_ca_path.empty();
         }
         if (config.has("security.cert_path"))
         {
-            new_cert_path = config.getString("security.cert_path");
-            miss_cert_path = false;
+            new_cert_path = Poco::trim(config.getString("security.cert_path"));
+            miss_cert_path = new_cert_path.empty();
         }
         if (config.has("security.key_path"))
         {
-            new_key_path = config.getString("security.key_path");
-            miss_key_path = false;
+            new_key_path = Poco::trim(config.getString("security.key_path"));
+            miss_key_path = new_key_path.empty();
```

Comment:
The current behavior is if part of ca/cert/key path is missing, it will throw invalid config parameter error(https://github.com/pingcap/tiflash/pull/9234/files#diff-aba70a04f427b9688d370af9af31421048642976333ff681ac5fa66431a97e92R294), is this still the expected behavior after this pr?

Reply:
Yes, I've verified that those test case will run into the `catch ...` branch in the ut
https://github.com/pingcap/tiflash/blob/dacbd35f1c705045be0579e04cc79867478a904c/dbms/src/Common/tests/gtest_tiflash_security.cpp#L85-L122